src/Hl7.Fhir.Serialization.R4B.Tests/Hl7.Fhir.Serialization.R4B.Tests.csproj

```diff
@@ -4,7 +4,12 @@
 
 	<Import Project="..\firely-net-sdk-tests.props" />
 
-	<PropertyGroup>
+  <!-- Compiler settings -->
+  <PropertyGroup>
+    <LangVersion>11</LangVersion>
```

Comment:
You cannot do this without also updating the build scripts. But I also want to use C# 11, so we should.

Reply:
I have enabled C# 11 in the mainline develop, so this should no longer be necessary.